api/v1/qdrantcluster_types.go

```diff
@@ -146,6 +143,12 @@ type QdrantClusterSpec struct {
 	// This helps sharded but not replicated clusters to reduce downtime to possible minimum during restart.
 	// +optional
 	RestartAllPodsConcurrently bool `json:"restartAllPodsConcurrently,omitempty"`
+	// If Emit Sleep is set, an additional 'sleep <value>' will be emitted to the pod startup.
+	// The sleep will be added when a pod is restarted, it will not force any pod to restart.
+	// This feature is for debugging the core, e.g. if a pod is in crash loop, it provided a way
+	// to inspect the attached storage.
+	// +optional
+	EmitSleep *int `json:"emitSleep,omitempty"`
```

Comment:
We can improve the naming to better indicate what it does. Please consider renaming it to `initialStartupDelay` or `initialStartupDelaySeconds`.

Reply:
If it is for debugging purposes only, I would suggest that we go with an annotation instead.